cfe_internal/enterprise/federation/federation.cf

```diff
@@ -361,6 +366,8 @@ bundle agent entry
 bundle agent setup_status
 {
 @if minimum_version(3.11)
```

Comment:
@olehermanse I see that `bundle agent setup_status` is called for any case where federated reporting is `enabled`.  If we want to support 3.10 feeders this needs to be re-worked to not use the `with` attribute (introduced in 3.11).

Do you want me to open a separate PR or push to your fork or ...?

Reply:
Well, here is a PR against your PR.
https://github.com/olehermanse/masterfiles/pull/1

Reply:
Thanks!